README.md

```diff
@@ -21,7 +21,13 @@ $ ./capture.sh
 
 When you want to make a break just stop the process using `ctrl-c` and restart when you're done.
 
-Default capture rate is one screenshot every 4 seconds.
+Default capture rate is one screenshot every 4 seconds. This can however be adjusted with the --period command line option.
+
+By default, the screen will be captured regardless of which app you have in focus. When you switch from your Fusion360 session to Music to choose another song, this will be captured. This behaviour can be adjusted with the --app command line option. By providing the name of an app, screen captures will only be made when that specific app is in focus. An example with all options is shown bellow. The --app option is only available on macOS.
```

Comment:
Nit: options format

```suggestion
By default, the screen will be captured regardless of which app you have in focus. When you switch from your Fusion360 session to Music to choose another song, this will be captured. This behaviour can be adjusted with the `--app` command line option. By providing the name of an app, screen captures will only be made when that specific app is in focus. An example with all options is shown bellow. The `--app` option is only available on macOS.
```